### DISCUSSION TITLE
How to handle multi-threading in Toga

### DISCUSSION THREAD

**Discussion by johan-n-ictech** [beeware/toga#2175] [Debugging help]
Hi,

Just started to try out Toga. For my example program I want to receive messages from a server in 1 thread and display them in the main GUI. To start with a label is fine but later on I would like to display images and logs. 

I cannot find any good reference or example of how this can or should be done. Can you point me in the right direction?

Best Regards,
Johan

**Comment by mhsmith** [comment]
There are some examples [here](https://stackoverflow.com/a/77350586).

**Reply by johan-n-ictech** [reply]
Thanks! Seems to work.
